packages/salesforcedx-utils-vscode/src/cli/commandExecutor.ts

```diff
@@ -22,9 +22,9 @@ export interface CancellationToken {
 
 export class CliCommandExecutor {
   private readonly command: Command;
-  private readonly options: ExecOptions;
+  private readonly options: SpawnOptions;
```

Comment:
This is a legit typing issue that we need to fix. So far it has not affected us since we don't use any of the other options.